include/cuco/detail/pair/pair.inl

```diff
@@ -50,10 +50,6 @@ __host__ __device__ constexpr bool operator==(cuco::pair<T1, T2> const& lhs,
 
 }  // namespace cuco
 
-namespace thrust {
-#include <cuco/detail/pair/tuple_helpers.inl>
-}  // namespace thrust
```

Comment:
@sleeepyjack

Just a reminder to ourselves, `thrust::pair` is the same as `cuda::std::pair` so the current pair trait logic can be simplified.